monitor/monitor.go

```diff
@@ -143,6 +144,9 @@ func (m *Monitor) Start(baseHeight uint32) {
 	} else if !exists {
 		startHeight = baseHeight
 	} else {
+		if dbHeight > math.MaxUint32 {
```

Comment:
seems in some places we use `math.MaxUint32` and in other places we use `^uint32(0)`. Are they the same? If so, should we keep using one?